code/game/atoms_movable.dm

```diff
@@ -4,6 +4,7 @@
 #if DM_VERSION >= 512
 #error HEY, PIXEL_SCALE probably exists now, remove this gross ass shim.
 #endif
+#endif

```

Comment:
So it just looks instead like the ref is shit, not explicitly saying that ```#endif```s are required, instead suggesting that any ```#else```,```#elseif```, or ```#endif``` is enough (and certainly not stating that one is require per instance)